src/input-select/input-select.tsx

```diff
@@ -61,7 +61,7 @@ export const InputSelect = <T, V>({
     const handleSelectorClick = (event: React.MouseEvent) => {
         event.preventDefault();
 
-        if (!disabled) {
+        if (!disabled && !otherProps.readOnly) {
```

Comment:
I somehow find this condition abit awkward. There is no issue and it works but maybe we can be abit more explicit by reversing the condition
```
if (disabled || otherProps.readOnly) {
    return;
}
```